### DISCUSSION TITLE
Step up authentication.

### DISCUSSION THREAD

**Discussion by reduno85** [OpenIDC/mod_auth_openidc#837] [Q&A]
Hello ,

I am facing a problem when using the stepup authentication  only if am already logged in with a lower auth (username/password): 

<img width="895" alt="Screenshot 2022-04-11 at 15 39 49" src="https://user-images.githubusercontent.com/56034187/162752065-83f8718b-5a0f-4d3b-a9de-3f79674103c9.png">

<img width="1397" alt="Screenshot 2022-04-11 at 15 35 43" src="https://user-images.githubusercontent.com/56034187/162751363-34148ad9-2d77-4445-adb3-15e6854ce84d.png">

looks like some  retries but no response 
does someone ever faced something like this 
Thanks and Best Regards

**Comment by zandbelt** [comment]
the server debug logs will tell you what is going on

**Reply by reduno85** [reply]
thanks for the reply, what i get is this many times   : 

[Mon Apr 11 19:33:08.262588 2022] [authz_core:error] [pid 24461:tid 140262458210048] [client [10.30.23.4:22369](http://10.30.23.4:22369/)] AH01631: user [username@hostname:443/auth/oauth2/realms/root/realms/intranetb2x](mailto:username@hostname:443/auth/oauth2/realms/root/realms/intranetb2x): authorization failure for "/strong":

But I am not even reaching the login page of the OP.

**Reply by reduno85** [reply]
I can see also this [Tue Apr 12 11:10:50.345728 2022] [ssl:info] [pid 24725:tid 140699286624000] (104)Connection reset by peer: [client [10.30.23.2:18145](http://10.30.23.2:18145/)] AH01991: SSL input filter read failed.

Do you please have any idea why this would happen? I have 2 apache server instances behind a load balancer.

**Reply by zandbelt** [reply]
are you using a shared caching mechanism for the sessions (`OIDCCacheType`)? also, which version of the module are you using?

**Reply by reduno85** [reply]
I am not using any caching for the session and the version  I am using  is 2.4.11

**Reply by zandbelt** [reply]
please provide the full server debug logs leading up to the error

**Reply by reduno85** [reply]
Sure here are they  I did lower the log level to debug to see al what happens : 
[logs.txt](https://github.com/zmartzone/mod_auth_openidc/files/8480590/logs.txt)

**Reply by zandbelt** [reply]
that looks ok to me: the server sends an HTML response with an auto-refresh that should trigger the redirect to the Provider; probably there's a browser setting interfering with this (e.g. IE security?)

**Reply by reduno85** [reply]
Not really I am using chrome. This only happens when I have a mod_auth_openidc_session. when deleted it works and redirects to the login page but in this case tI do not think the OIDCUnAutzAction is not triggered or ?

**Reply by zandbelt** [reply]
it is triggered but the browser somehow does not handle the refresh page; I've tested your config locally and it works; I'm afraid there's not much more that I can do here; you could get commercial support to debug your setup via sales@zmartzone.eu
